content/en/docs/access-control-and-account-management/external-authentication/set-up-external-authentication.md

```diff
@@ -92,15 +84,15 @@ You need to deploy a Kubernetes cluster and install KubeSphere in the cluster. F
      iam.kubesphere.io/origin-uid: <Third-party username>
    ```
    
-4. After the fields are configured, run the following command to restart ks-apiserver.
+4. After the fields are configured, save your changes, and run the following command to restart ks-installer.
```

Comment:
No need to restart the ks-installer pod. Ks-Installer watches the CRD changes to trigger a re-run. Just need to wait until it runs successfully.